auth0/resource_auth0_prompt_test.go

```diff
@@ -8,7 +8,6 @@ import (
 )
 
 func TestAccPrompt(t *testing.T) {
-
```

Comment:
**Question (non-blocking):** Is there some way you can apply these formatting rules to the entire codebase? I'm finding that these formatting rules make the PRs diffs quite noisy

Reply:
Yep, there definitely is, however I opted to do this in this PRs as I was going through each resource one by one and make sure each PR has at least 2 commits:
- [Refactor prompt resource](https://github.com/auth0/terraform-provider-auth0/pull/112/commits/8d9c9132300ea41081014437d7d6fd735faed0ee) 1 focused just on code style and small refactors
- [Stop ignoring errors when setting resource data within prompts](https://github.com/auth0/terraform-provider-auth0/pull/112/commits/9be4add764c96d22d509ac120fea99d8d3529d0a) one with the actual focus for the ticket

After touching all of them through our error handling epic I plan to introduce a linter so we don't have to do this any more and fix everything else that might pop in 1 go. 

Also I'm not opposed to remove the code style changes to ease with reviewing if needed. Just let me know:)